projects/ikia/seeds/questionnaires/ikia.rb

```diff
@@ -0,0 +1,257 @@
+# frozen_string_literal: true
+
+db_title = '' # Dagboekvragenlijst moet geen titel hebben alleen een logo
+
+db_name1 = 'demo'
```

Comment:
dit klopt?

Reply:
Ik pas het wel aan.

Reply:
Volgens mij maak je die demo questionnaire alleen aan als je de demo organisatie hebt, maar voor onze eigen sanity is het misschien wel een goed idee om questionnaire namen uniek te houden.

Reply:
Fixed.